source/docs/v3/configuration.blade.md

```diff
@@ -60,10 +60,8 @@ The list of domains that host your [central app]({{ $page->link('the-two-applica
 This config array lets you enable, disable or add your own [tenancy bootstrappers]({{ $page->link('tenancy-bootstrappers') }}).
 
 ### Database {#database}
-> Note: If you're using [Laravel Sail](https://laravel.com/docs/9.x/sail), ensure that `DB_USERNAME` has the necessary permissions. You can grant privileges to users by running `GRANT ALL PRIVILEGES on *.* to 'sail'@'%';` in the MySQL console. 
-> Once you have finalized the permissions, always be sure to reload all the privileges `FLUSH PRIVILEGES;` 
 
-`tenancy.database.*`
+>If you're using Laravel Sail, please refer the [Laravel Sail integration guide]({{ $page->link('integrations/sail') }}):
```

Comment:
```suggestion
> If you're using Laravel Sail, please refer the [Laravel Sail integration guide]({{ $page->link('integrations/sail') }}):
```
We use this code style for quotes, `>foo` isn't valid markdown even though it works here. Please update it everywhere.

Also, what's the `:` at the end of the sentence for?

Reply:
removed `:`, used italics.

Reply:
You can use `>`, we use it often, but it's meant to be followed by a space.

Reply:
Your wish is my command, Please let me know how you want it. I will change accordingly. I feel italics should be fine

Reply:
Just use `> this`

Reply:
Done